LucaNyckees/climate-forecasting-v4
# 1 extensive code cleaning and refactoring

### Description

Updated most of the project by
- cleaning code
- installing venv
- making requirements
- implementing good paths practise
- refactoring methods
- re-organizing folder structure
- ...

### To test

Run the command `streamlit run src/streamlit_app.py` and check everything works fine.

### Note

Notebooks are still not cleaned and may present errors when running them.
